frontend/src/index.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <BrowserRouter>
     <App />
-  </React.StrictMode>,
+  </BrowserRouter>,
```

Comment:
Why remove `StrictMode`?

Reply:
Got a "Error: Invariant failed: You should not use <Switch> outside a <Router>". But we can put the StrictMode around the router like
```
  <React.StrictMode>
      <BrowserRouter>
        <App />
      </BrowserRouter>
  </React.StrictMode>,
```
Anything wrong with this approach?

Reply:
Looking at `StrictMode` docs closer, looks like it just applies everything to decadents. Added it back in.